lib/linter/config-comment-parser.js

```diff
@@ -139,7 +139,7 @@ module.exports = class ConfigCommentParser {
         const items = {};
 
         string.split(",").forEach(name => {
-            const trimmedName = name.replace(/^\s*(?<quote>['"]?)(?<ruleId>\S*)\k<quote>\s*$/u, "$<ruleId>");
+            const trimmedName = name.trim().replace(/^(?<quote>['"]?)(?<ruleId>.*)\k<quote>$/us, "$<ruleId>");
```

Comment:
Can we also add some tests that have the whitespace @mdjermanovic mentioned to verify that this is the correct fix?

Reply:
I added a test case to tests/lib/linter/config-comment-parser.js 👍